### PR TITLE
Update package.json

### PR DESCRIPTION
**Pull request by JMVHill**

### CODE REVIEW

**Comment by rjmcneill** [comment]
LGTM :+1:

File: package.json [hunk 1]
```diff
@@ -46,7 +46,7 @@
   "keywords": [
     "Scuttling"
   ],
-  "author": "JArnstein",
+  "author": "JArnstein (c) 2015",
```
Comment (jdhodges) [inline package.json]: How about:
`"author": "JArnstein © 2015",`

I feel the current implementation is sub-optimal

Comment (stevenjob) [inline package.json]: Agree 100%

Comment (jdhodges) [inline package.json]: One more thing, surely it should be 2016?

Comment (rjmcneill) [inline package.json]: 2016 then as well?
If there's a new PR update coming may as well address both issues

Comment (jdhodges) [inline package.json]: So in summary, 100% of the changes submitted here are wrong. Let me know when you've fixed it.

File: package.json [hunk 2]
```diff
@@ -46,7 +46,7 @@
   "keywords": [
     "Scuttling"
   ],
-  "author": "JArnstein (c) 2015",
+  "author": "JArnstein © 200BC to (new Date().getFullYear() + 1)",
```
Comment (jdhodges) [inline package.json:49]: This is a string, therefore `new Date().getFullYear() + 1` does not evaluate to anything.

Comment (stevenjob) [inline package.json:49]: Not TR quality

**Comment by jdhodges** [comment]
"Closed" is being kind. This PR was abandoned.

**Pull request closed without merging**